<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPanel</class>
 <widget class="QWidget" name="DataPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCurrentVes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>SEV:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxCurrentVes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxVesData">
     <property name="title">
      <string>Datos del SEV</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonField">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Campo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonSplice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Empalmes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButtonCalculated">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Calculados</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonModeled">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Modelados</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBoxVesModel"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonVfsa">
       <property name="text">
        <string>V.F.S.A.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZohdy">
       <property name="text">
        <string>Zohdy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonField</sender>
   <signal>clicked()</signal>
   <receiver>DataPanel</receiver>
   <slot>changeShowedData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSplice</sender>
   <signal>clicked()</signal>
   <receiver>DataPanel</receiver>
   <slot>changeShowedData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCalculated</sender>
   <signal>clicked()</signal>
   <receiver>DataPanel</receiver>
   <slot>changeShowedData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonModeled</sender>
   <signal>clicked()</signal>
   <receiver>DataPanel</receiver>
   <slot>changeShowedData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>comboBoxCurrentVESIndexChanged(int)</signal>
  <slot>changeShowedData()</slot>
 </slots>
</ui>
